src/main/java/codeu/model/data/ProfilePage.java

```diff
@@ -0,0 +1,54 @@
+package codeu.model.data;
+
+import java.util.ArrayList;
+import java.util.UUID;
+
+public class ProfilePage {
+	
+	String aboutMe;
```

Comment:
Should we make these variables private?

Reply:
I was wondering the same thing.  I thought that if we were checking in some doPost if the User was capable of changing something then it would be safe but I don't really know.  Evan?

Reply:
@JudahBrick Not sure what you're asking exactly, could you send me the design doc so I can take a look at the flow for the ProfilePage?

Reply:
After looking over the design doc, yes all of them should be private. In Java we aim to have as little visibility for our methods and variables as possible.

You also already have getters and setters for the variables you want to be able to change, so we're all set on being able to access the variables we want.

On another note, do we want to be able `setName` via the ProfilePage? This could cause disparity between the ProfilePage and the User, where name is originally stored.